integration_tests/Cargo.toml

```diff
@@ -24,6 +24,9 @@ debug = false
 path = "../rusoto/core"
 default-features = false
 
+[dependencies.rusoto_credential]
+path = "../rusoto/credential"
```

Comment:
I think we can skip this change, since [rusoto_core reexports rusoto_credential](https://github.com/rusoto/rusoto/blob/master/rusoto/core/src/lib.rs#L34).